### PR TITLE
Added error handling

### PR DESCRIPTION
**Pull request by gecayonK3nch**

### CODE REVIEW

**Review by danegumya — APPROVED** [review]
very good

File: main.py [hunk 1]
```diff
@@ -29,7 +29,11 @@ def run():
 
 @app.get("/get")
 def get():
-    return {"response": json.dumps(run())}
+    result = run()
+    if result['status'] == 'success':
+        return result['response']
+    else:
+        return result['message'][:1488]
```
Comment (SemyonHoyrish) [inline main.py]: Why we cut first 1488 characters?

**Review by SemyonHoyrish — CHANGES_REQUESTED** [review]
do not use magic number

**Comment by Rogon1256** [comment]
approve?

**Review by SemyonHoyrish — APPROVED** [review]

**Review by danegumya — APPROVED** [review]
not bad but 1488 was better

**Review by egor-abramov — APPROVED** [review]
OK

**Comment by gecayonK3nch** [comment]
> not bad but 1488 was better

1488 was saved. Magic Number was removed.

**Review by Rogon1256 — APPROVED** [review]
amogus

**Pull request merged**